chart/compass/charts/operations-controller/templates/crds.yaml

```diff
@@ -33,10 +33,14 @@ spec:
       description: Operation is the Schema for the operations API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
I think this line breaks will not work.
To make a multiline property in yaml you should use ">-" like this:
```
description: >-
   'APIVersion defines the versioned schema of this representation
   of an object. Servers should convert recognized schemas to the latest
   internal value, and may reject unrecognized values. More info:
   https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```